projects/core/src/accordion/accordion-header.element.scss

```diff
@@ -26,6 +26,12 @@
   font-size: var(--font-size);
   font-weight: var(--font-weight);
   color: var(--color);
+  box-shadow: var(--box-shadow);
```

Comment:
These CSS variables must be initialized to the default values in the `:host` block. Leaving them uninitialized on the host will result in any definition higher in the DOM cascading into this component (any `--box-shadow` on an ancestor will be reflected here). 

Can you add a comment block to the top of this file indicating these three new values enable the accordion to be styled like the ng-clarity stepper? Moving forward, it will be helpful to have that context readily available when we need to make changes.